packages/manager/src/services/feedback/queries.tsx

```diff
@@ -0,0 +1,47 @@
+import { feedbackReq } from '@app/config/feedbackReq';
+import { useMutation } from '@tanstack/react-query';
+
+const body = (variables: any) => {
+  const query = `
+    mutation (
+      $projectId: String
+      $summary: String
+      $experience: String
+      $category: FeedbackCategory
+      $stackInfo: StackInfoInput
+      $userId: String
+      $error: String
+    ) {
+      createFeedback(
+        input: {
+          projectId: $projectId
+          summary: $summary
+          experience: $experience
+          category: $category
+          createdBy: $userId
+          stackInfo: $stackInfo
+          error: $error
+        }
+      ) {
+        summary
+        ticketUrl
+        error
+        stackInfo {
+          path
+          stack
+        }
+      }
+    }
+  `;
+  return JSON.stringify({
+    query,
+    variables
+  });
+};
+
+const createFeedback = async (variables: any): Promise<any> => {
```

Comment:
Please use the appropriate type here `async (variables: any): Promise<any>` in place of any.

Reply:
fixed